docs/tutorials/secrets.rst

```diff
@@ -29,11 +37,16 @@ Then, you can read and set secrets within a script or a Python REPL:
     :dedent: 8
 
 
-``name`` is the secret identifier. It can be any string you like, as long as it fits in the length limit. This identifier is subsequently used to access the secret. Secrets are scoped to a single user account, so there's no risk you're overriding somebody else's entry.
+``name`` is the secret identifier.
+Secret names may include letters, numbers, and dashes, and must not start or end with a dash.
+This identifier is subsequently used to access the secret.
+Secrets are scoped to a single user account, so there's no risk you're overriding somebody else's entry.
```

Comment:
This isn't true, secrets are scoped to the workspace (so you could delete and recreate someone else's secret if you're in a shared workspace)

Reply:
Good catch! New version now reads:

```
``name`` is the secret identifier.
Secret names may include letters, numbers, and dashes, and must not start or end with a dash.
This identifier is subsequently used to access the secret.
Secrets are scoped to a workspace, so care must be taken when managing secrets in shared workspaces.
```

Reply:
Thank you!